openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java

```diff
@@ -1187,7 +1186,7 @@ private String makeSureImported(String typeName, Set<String> imports) {
     public String toVarName(String name) {
         var varName = super.toVarName(name);
 
-        if (varName.chars().allMatch(Character::isUpperCase)) {
+        if (varName.chars().allMatch(c -> Character.isUpperCase(c) || c == '_')) {
```

Comment:
why is this change necessary in the PR to update versions?

Reply:
because this fix for openapi-generator 7.5.0, without this fix you can't use 7.5.0